src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -60,16 +60,13 @@ public static int IndexOfAny(ref byte searchSpace, int searchSpaceLength, ref by
             if (valueLength == 0)
                 return 0;  // A zero-length sequence is always treated as "found" at the start of the search space.
 
-            int index = -1;
+            int index = int.MaxValue;
             for (int i = 0; i < valueLength; i++)
             {
                 var tempIndex = IndexOf(ref searchSpace, Unsafe.Add(ref value, i), searchSpaceLength);
-                if (tempIndex != -1)
-                {
-                    index = (index == -1 || index > tempIndex) ? tempIndex : index;
-                }
+                index = (tempIndex == -1 || tempIndex > index) ? index : tempIndex;
```

Comment:
You can use the same leaner pattern here as in LastIndexOfAny. Just need to use unsigned comparison:

`index = ((uint)index > (uint)tempIndex) ? index : tempIndex;`

Reply:
`index = ((uint)tempIndex > (uint)index) ? index : tempIndex;`
is correct.